db/external_sst_file_ingestion_job.cc

```diff
@@ -95,6 +95,14 @@ Status ExternalSstFileIngestionJob::Prepare(
         "behind mode.");
   }
 
+  // Overlapping files need at least two different sequence numbers. If settings
+  // disables global seqno, ingestion will fail anyway, so fail fast in prepare.
+  if (!ingestion_options_.allow_global_seqno && files_overlap_) {
+    return Status::InvalidArgument(
+        "Global seqno is required, but disabled (because external files key "
```

Comment:
This error message is a bit unusual. I'm trying to keep this parsing logic continue to work: https://github.com/facebook/mysql-5.6/blob/fb-mysql-8.0.32/storage/rocksdb/rdb_sst_info.cc#L511